packages/calcite-components/src/components/shell/shell.tsx

```diff
@@ -53,6 +54,8 @@ export class Shell implements ConditionalSlotComponent {
 
   @State() hasModals = false;
 
+  @State() hasSheets = false;
```

Comment:
nitpick: can this be `hasSheet` or `hasSlottedSheet` instead of `hasSheets`.

Reply:
I'd like to maintain consistency with the above `hasModals`. It can be multiple sheets.

Reply:
makes sense. wasn't aware that users will slot multiple sheets.